src/Components/Icon/Icon.tsx

```diff
@@ -0,0 +1,24 @@
+import { Facebook, Twitter, Google, GitHub } from "@mui/icons-material";
+
+import { SvgIconProps } from "@mui/material";
+
+type SocialMediaIconProps = {
+  type: string;
+} & SvgIconProps;
+
+const Icon = ({ type, ...props }: SocialMediaIconProps) => {
```

Comment:
Rename this component to SocialMediaIcon.

Reply:
what if you want to use just icon?

Reply:
SocialMediaIcon is a type of an Icon component. It should reside in Icon folder but with a different name.
Icon component is separate component